### PR TITLE
base/: Add CPEs

### PR DESCRIPTION
**Pull request by ReillyBrogan**
This adds CPEs for all packages under base.

### CODE REVIEW

File: base/dosfstools/monitoring.yml [hunk 1]
```diff
@@ -1,2 +1,4 @@
 releases:
   id: 455
+security:
+  cpe: cpe:2.3:a:dosfstools_project:dosfstools:*:*:*:*:*:*:*:*
```
Comment (der-eismann) [inline base/dosfstools/monitoring.yml]:
```suggestion
  cpe: cpe:2.3:a:dosfstools_project:dosfstools
```

File: base/curl/monitoring.yml [hunk 2]
```diff
@@ -1,2 +1,4 @@
 releases:
   id: 381
+security:
+  cpe: cpe:2.3:a:curl:curl:*:*:*:*:*:*:*:*
```
Comment (der-eismann) [inline base/curl/monitoring.yml]:
```suggestion
  cpe: cpe:2.3:a:curl:curl
```

**Review by der-eismann — APPROVED** [review]
Awesome, thank you!

**Comment by ermo** [comment]
Please change `monitoring.yml` file format to ensure scalability (per [this forum thread](https://forums.serpentos.com/d/47-security-monitoring-file-format-discussion)):

```YAML
security:
    vendor: somevendor
    product: someproduct
    ignore:
        - CVE-x
        - CVE-y
        - CVE-z
```

**Comment by ikeycode** [comment]
Amazing effort all round. Nice one!

**Pull request merged**